lib/screens/album_screen.dart

```diff
@@ -46,23 +46,35 @@ class _AlbumScreenState extends State<AlbumScreen> {
 
             if (isOffline) {
               final isarDownloads = GetIt.instance<IsarDownloads>();
-              albumScreenContentFuture ??= isarDownloads.getCollectionSongs(parent);
+              //TODO make a reasonable API for this
+              albumScreenContentFuture ??= Future.wait([
+                isarDownloads.getCollectionSongs(parent, playable: false),
+                isarDownloads.getCollectionSongs(parent, playable: true)
+              ]);
```

Comment:
Introducing a custom type for this (like suggested in a previous comment) should improve the API. Just pass the whole collection and do some conditional rendering in `AlbumScreenContent`